feature-pack/src/main/resources/modules/system/layers/base/org/jboss/weld/core/main/module.xml

```diff
@@ -51,5 +51,10 @@
         <module name="org.jboss.weld.spi" />
         <module name="org.jboss.logging" />
         <module name="sun.jdk" optional="true" />
+        <system export="false">
+            <paths>
+                <path name="com/sun/org/apache/bcel/internal/classfile"/>
```

Comment:
I thought there was some reason to limit the set of packages in sun.jdk module...